full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -157,23 +152,18 @@ pub enum JsonCommandRequest {
     get_account_status {
         account_id: String,
     },
-    get_address_for_account {
-        account_id: String,
-        index: i64,
+    get_accounts {
+        offset: Option<u64>,
```

Comment:
omg yes! I always wondered why offset and limit were strings

Reply:
same!